ui/src/MobileClientServiceChart.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { DonutChart } from 'patternfly-react';
+
+import './styles/MobileClientServiceChart.css'
+
+class MobileClientServiceChart extends Component {
+    viewAllServices(){
```

Comment:
This is not really part of this component. We can move it when we integrate all the components though